src/sagemaker/chainer/estimator.py

```diff
@@ -126,22 +130,18 @@ def __init__(
             :class:`~sagemaker.estimator.Framework` and
             :class:`~sagemaker.estimator.EstimatorBase`.
         """
-        if framework_version is None:
+        validate_version_or_image_args(framework_version, py_version, image_name)
+        if py_version == "py2":
             logger.warning(
-                empty_framework_version_warning(defaults.CHAINER_VERSION, self.LATEST_VERSION)
+                python_deprecation_warning(self.__framework_name__, defaults.LATEST_PY2_VERSION)
             )
```

Comment:
I don't think Chainer needs this since we're not planning on releasing any new versions without Python 2

edit: ...oh. I see it's copied from below. well, nvm then hahaha.